Exercise.md

```diff
@@ -0,0 +1,5 @@
+# New-Repo-Rocio
+what app you want to use
```

Comment:
is this something to display? something to store? just to have this in mind, it would be clearer if you think which is an output for the user, and which inputs you will request for the user